sdk/python/feast/infra/offline_stores/bigquery.py

```diff
@@ -478,6 +490,39 @@ def persist(self, storage: SavedDatasetStorage):
     def metadata(self) -> Optional[RetrievalMetadata]:
         return self._metadata
 
+    def supports_remote_storage_export(self) -> bool:
+        return True
+
+    def to_remote_storage(self) -> List[str]:
+        if self.config.offline_store.gcs_staging_location or not self._gcs_path:
```

Comment:
shouldn't this check just be `if not self._gcs_path`, since that is initialized in `__init__` using `self.config.offline_store.gcs_staging_locatio`